### PR TITLE
add in st_build.py

### PR DESCRIPTION
**Pull request by kwyatt**
add in st_build.py

### CODE REVIEW

File: st_build.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    if (tc_conf and tc_conf.lower().find('win64') != -1):
+      plat = 'win64'
+  print "st_build_android.sh %s %s" % (plat, args.arch)
+  exit(os.system(path.join(scriptdir, "st_build_android.sh %s %s" % (plat, args.arch))))
+elif args.ios:
+  print "st_build_ios.sh"
+  exit(os.system(path.join(scriptdir, "st_build_ios.sh")))
+elif (windows):
+  print "st_build_32.bat"
+  exit(os.system(path.join(scriptdir, "st_build_32.bat")))
```
Comment (sparist) [inline st_build.py]: I think this should be st_build_win32.bat

File: st_build.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    print >> sys.stderr, "Must specify arch when compiling for Android"
+    exit(-1)
+  if osx:
+    plat = 'osx'
+  elif linux:
+    plat = 'linux'
+  else:
+    plat = 'win32'
+    if (tc_conf and tc_conf.lower().find('win64') != -1):
+      plat = 'win64'
```
Comment (sparist) [inline st_build.py]: I don't think we need to support building Android on Windows. The OpenSSL scripts don't support it right now; only "osx" and "linux"

File: st_build.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  exit(os.system(path.join(scriptdir, "st_build_android.sh %s %s" % (plat, args.arch))))
+elif args.ios:
+  print "st_build_ios.sh"
+  exit(os.system(path.join(scriptdir, "st_build_ios.sh")))
+elif (windows):
+  print "st_build_32.bat"
+  exit(os.system(path.join(scriptdir, "st_build_32.bat")))
+elif (osx):
+  print "st_build_osx.sh"
+  exit(os.system(path.join(scriptdir, "st_build_osx.sh")))
```
Comment (sparist) [inline st_build.py]: Could probably add an "else error" at the end of this, if for some reason there is a problem getting the platform (e.g. the TeamCity build name is changed to something that doesn't include the platform)

**Comment by kwyatt** [comment]
Made recommended changes

File: st_build.py [hunk 4]
```diff
@@ -38,18 +38,17 @@
     plat = 'osx'
   elif linux:
     plat = 'linux'
-  else:
```
Comment (sparist) [inline st_build.py]: This could be an "else error" case

Comment (sparist) [inline st_build.py]: This could be an "else error" case to catch build config misnomers

**Pull request merged**